Add ResourceQuota propagation to the default setting

bells17 opened the pull request.

ymmt2005 commented on docs/config.md:
```diff
@@ -54,34 +54,6 @@ controller:
 ## ClusterRoleBindings
 
-A built-in ClusterRole `admin` is bound by default to allow `accurate-controller` to watch and propagate namespace-scope resources.  However, `admin` does not contain verbs for [ResourceQuota][] and may not contain custom resources.
+A built-in ClusterRole `admin` is bound by default to allow `accurate-controller` to watch and propagate namespace-scope resources.
```
What about ResourceQuota?

ymmt2005 commented on controllers/subnamespace_controller.go:
```diff
@@ -29,6 +29,7 @@ type SubNamespaceReconciler struct {
 //+kubebuilder:rbac:groups=accurate.cybozu.com,resources=subnamespaces/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=accurate.cybozu.com,resources=subnamespaces/finalizers,verbs=update
 //+kubebuilder:rbac:groups="",resources=namespaces,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups="",resources=resourcequotas,verbs=get;list;watch;create;update;patch;delete
```
The subnamespace controller does not require any privileges for ResourceQuotas.
That should be configured through a Helm variable.

bells17 commented:
Thank you @ymmt2005 .

> The subnamespace controller does not require any privileges for ResourceQuotas.
That should be configured through a Helm variable.

I removed ResourceQuotas RBAC manifests that were generated by kubebuilder comments and controller-tools and added ResourceQuotas RBAC manifests to default settings of additionalRBAC.
Does this match what you intended?

ymmt2005 commented:
> Don't you need to add ResourceQuota to watches in values.yaml?

I guess it needs to be added.

@bells17 
Could you add a test to watch and propagate ResourceQuota?
